README.md

```diff
@@ -189,22 +189,7 @@ C5.4XL	                        M6g.4XL
 ```
 
 ## Java
-For languages that rely on a JIT (such an Java), the symbol information that is
```

Comment:
Feels like a bit of intro text is needed to motivate multi-arch JARs and building them.

Reply:
PR updated with some justification as to why multi-arch Jars are beneficial.